README.md

```diff
@@ -193,6 +191,18 @@ If you specify an array of connections instead of a single connection object Ras
 }
 ```
 
+#### Channel pooling
+Rascal pools channels it uses for publishing messages. It creates a two pools per vhost - one for confirm channels, and other one for "regular" channels. Default maximum pool size for these is 1. If you think it is a limiting factor in your publishing code, the value can be changed through config.
+```json
+{
+  "vhosts": {
+    "v1": {
+      "channelPoolSize": 3
```

Comment:
Yep was thinking about it, but wasn't sure if added complexity is warranted, considering there are about 65k channel available, so I just stuck with one setting at the moment.

If we go for explicit two settings, would you just call them `channelPoolSize` and 'confirmChannelPoolSize`? Would you perhaps nest them under something like `publicationChannelPool`?